libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/notification/Notification.java

```diff
@@ -20,27 +20,24 @@
 
 public class Notification {
 
-    // Denotes notification type. MEMBERSHIP_APPROVAL, MEMBERSHIP_EXPIRY etc.
-    private String type;
-
     // Intended recipients of notification
     private Set<String> recipients;
 
     // key value pair describing additional details about notification
     private Map<String, String> details;
 
-    public Notification (String type, Set<String> recipients, Map<String, String> details) {
-        this.type = type;
+    // Utility class to convert the notification into an email
```

Comment:
NotificationToEmailConverter will be implemented inside each NotificationTask. Its purpose is to convert the notification into an email